src/components/Footer.tsx

```diff
@@ -74,7 +74,7 @@ export default function Footer({ lang }: { lang: string }) {
               </li>
             </ul>
           </div>
-          <div className={"col-span-2 sm:col-span-1"}>@ 2022, Indomptable</div>
+          <div className={"col-span-2 sm:col-span-1"}>@ 2024, Indomptable</div>
```

Comment:
@7H3XXX , i think you need to replace that value with new Date().getFullYear(), to definitively solve the problem of updating this value every year